drivers/flash/flash_mcux_flexspi_hyperflash.c

```diff
@@ -427,6 +427,9 @@ static int flash_flexspi_hyperflash_write(const struct device *dev, off_t offset
 		key = irq_lock();
 	}
 
+	(void)memc_flexspi_update_clock(data->controller, &data->config,
```

Comment:
Minor comment- the description for your commit "drivers: flash: fix hyperflash write operations" currently mentions disabling the instruction cache, but I don't see any calls to do that in the PR. Could you update the commit message?

Reply:
Thanks! I fixed it.